application/src/main/java/org/thingsboard/server/service/subscription/TbTimeSeriesSubscription.java

```diff
@@ -53,12 +53,8 @@ public TbTimeSeriesSubscription(String serviceId, String sessionId, int subscrip
     }
 
     @Override
-    public boolean equals(Object o) {
-        return super.equals(o);
+    protected boolean canEqual(final Object other) {
```

Comment:
Remove all code that is not related to the scope of changes. Create new PR and cover this with TESTS!!!